lib/lbt/bundle/Builder.js

```diff
@@ -563,18 +617,16 @@ function rewriteDefine(targetBundleFormat, code, moduleName, avoidLazyParsing) {
 			}
 
 			// console.log("rewriting %s", module, defineCall);
-
 			return ast;
 		}
 
 
 		if ( isMethodCall(ast.body[0].expression, CALL_SAP_UI_DEFINE) ) {
 			const defineCall = ast.body[0].expression;
-
 			// rewrite sap.ui.define to sap.ui.predefine
 			if ( defineCall.callee.type === Syntax.MemberExpression
 					&& defineCall.callee.property.type === Syntax.Identifier
-					&& defineCall.callee.property.name === "define" ) {
+					&& defineCall.callee.property.name === "define") {
```

Comment:
unbalanced. I understand that whitespace after/before parentheses is a matter of taste, but mixing both approaches does not qualify as a good compromise, IMHO :-)

Reply:
one word: prettier